triton/kubernetes/FIL/interact_with_triton.ipynb

```diff
@@ -0,0 +1,307 @@
+       "       [0.64918005, 0.35081992],\n",
+       "       [0.58112514, 0.4188749 ],\n",
+       "       ...,\n",
+       "       [0.51727444, 0.48272556],\n",
+       "       [0.8467938 , 0.15320626],\n",
+       "       [0.93336314, 0.06663687]], dtype=float32)"
+      ]
+     },
+     "execution_count": 15,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "result_http = request_http.as_numpy('output__0')\n",
+    "result_http"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### gRPC Client Example."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Is Server Ready: True\n",
+      "Is FIL model ready: True\n"
+     ]
+    }
+   ],
+   "source": [
+    "grpc_client = triton_grpc.InferenceServerClient(\n",
+    "    url=f'{host}:{grpc_port}',\n",
+    "    verbose = False\n",
+    ")\n",
+    "\n",
+    "while (not (grpc_client.is_server_ready() or grpc_client.is_model_ready('fil'))):\n",
+    "    print(\"Waiting on server ready\")\n",
+    "    time.sleep(5)\n",
+    "    \n",
+    "print(f\"Is Server Ready: {grpc_client.is_server_ready()}\")\n",
+    "print(f\"Is FIL model ready: {grpc_client.is_model_ready('fil')}\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data = numpy.random.rand(samples, features).astype('float32')\n",
+    "triton_input_grpc = triton_grpc.InferInput(\n",
+    "    'input__0',\n",
+    "    [samples, features],\n",
+    "    'FP32'\n",
+    ")\n",
+    "\n",
+    "triton_input_grpc.set_data_from_numpy(data)\n",
+    "triton_output_grpc = triton_grpc.InferRequestedOutput('output__0')\n",
+    "\n",
+    "request_grpc = grpc_client.infer(\n",
+    "    'fil',\n",
+    "    model_version='1',\n",
+    "    inputs=[triton_input_grpc],\n",
+    "    outputs=[triton_output_grpc]\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "array([[0.360883  , 0.639117  ],\n",
+       "       [0.6957687 , 0.30423132],\n",
+       "       [0.70181334, 0.2981867 ],\n",
+       "       ...,\n",
+       "       [0.82927394, 0.17072603],\n",
+       "       [0.43887025, 0.56112975],\n",
+       "       [0.28556466, 0.71443534]], dtype=float32)"
+      ]
+     },
+     "execution_count": 17,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "result_grpc = request_grpc.as_numpy('output__0')\n",
+    "result_grpc"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Compare Results"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "metadata": {},
+   "outputs": [
+    {
+     "ename": "AssertionError",
```

Comment:
Just making sure - is this assertion error expected?

Reply:
It was at the time, because it wasn't able to compare grpc and tcp approaches correctly. Should be fixed now.